diff --git a/etc/build/build.xml b/etc/build/build.xml
index 29035e5..a2e1ead 100644
--- a/etc/build/build.xml
+++ b/etc/build/build.xml
@@ -361,7 +361,7 @@
 
 	<target name="javac" depends="app" description="Java compilation">
 		<mkdir dir="${javac.dir}"/>
-		<javac srcdir="${src.dir}" excludes="**/EmbedServlet2.java" encoding="utf-8" destdir="${javac.dir}" source="1.7" target="1.7" debug="true" debuglevel="lines,vars,source">
+		<javac srcdir="${src.dir}" encoding="utf-8" destdir="${javac.dir}" source="1.7" target="1.7" debug="true" debuglevel="lines,vars,source">
 			<classpath refid="javac.class.path" />
 		</javac>
 		<copy todir="${javac.dir}/META-INF">
diff --git a/src/com/mxgraph/online/EmbedServlet2.java b/src/com/mxgraph/online/EmbedServlet2.java
index cced6ae..6eca5ed 100644
--- a/src/com/mxgraph/online/EmbedServlet2.java
+++ b/src/com/mxgraph/online/EmbedServlet2.java
@@ -35,7 +35,7 @@ import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.google.appengine.api.utils.SystemProperty;
+//import com.google.appengine.api.utils.SystemProperty;
 
 /**
  * Servlet implementation class OpenServlet
@@ -79,7 +79,7 @@ public class EmbedServlet2 extends HttpServlet
 	{
 		if (lastModified == null)
 		{
-			// Uses deployment date as lastModified header
+			/*// Uses deployment date as lastModified header
 			String applicationVersion = SystemProperty.applicationVersion.get();
 			Date uploadDate = new Date(Long
 					.parseLong(applicationVersion
@@ -88,12 +88,13 @@ public class EmbedServlet2 extends HttpServlet
 
 			DateFormat httpDateFormat = new SimpleDateFormat(
 					"EEE, dd MMM yyyy HH:mm:ss z", Locale.US);
-			lastModified = httpDateFormat.format(uploadDate);
+			lastModified = httpDateFormat.format(uploadDate);*/
+			lastModified = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss z", Locale.US).format(new Date());
 		}
 
 		initLibraries(libraries);
 	}
-
+	
 	/**
 	 * Sets up collection of stencils
 	 */
